src/index.html

```diff
@@ -13,6 +13,51 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="display">
+      <div>
```

Comment:
Remember about correct indentation between parent and child elements. The `div` element on line 17 should be indented with 2 spaces as it is a child of the `div` element on line 16.